tests/outputs/grantreport/billinge_grant_report.txt

```diff
@@ -51,32 +51,17 @@ Key outcomes or Other achievements:
 3. Presentations at conferences and seminars
 
 * What do you plan to do during the next reporting period to accomplish the goals?
+#############################################################
```

Comment:
this looks like copy-pasted text....maybe assign this string to a variable and use that?